src/slotted_page.rs

```diff
@@ -837,7 +862,7 @@ mod tests {
         let page_size = SlottedPageMmap::new_page_size_for_value(value_size);
         assert_eq!(
             page_size,
-            128 + SlotHeader::size_in_bytes() + SlotHeader::size_in_bytes()
```

Comment:
Thank you very much for finding and fixing this bug!
I even wrote the broken unit test to go with it :face_with_head_bandage: